draft-ietf-scitt-architecture.md

```diff
@@ -895,78 +897,65 @@ By analyzing the relationship between data stored in the Transparency Service an
-The respective Attestation Result, for example, can show that the remote attestation Evidence was appraised by a Relying Party and complies with well-known Reference Values and Endorsements.
+Issuers and Transparency Services SHOULD:
 
-Auditors should be aware that the certification path information included in an unprotected `x5chain` header of a to-be-registered Signed Statement can be tampered with by a malicious Transparency Service (e.g., one that does not incorporate remote attestation), which may replace the intermediate certificates and ultimately connect to an unexpected root.
-This modification helps protect against person-in-the-middle attacks, but not denial-of-service.
-Auditors MUST perform certification path validation in accordance with PKIX rules specified in {{-PKIX}}.
-Auditors MUST verify that certification paths chain to one or more trust anchors (often represented as root certificates).
+- protect against malicious or vulnerable replicas by replicating with a consensus algorithm, such as Practical Byzantine Fault Tolerance {{PBFT}}.
+- provide additional authenticity assurances about its secure implementation and operation, enabling remote attestation of the hardware platforms and/or software Trusted Computing Bases (TCB) that run the Transparency Service.
 
-## Security Guarantees
 
-SCITT provides the following security guarantees:
 
-1. Statements made by Issuers about supply chain Artifacts are identifiable, can be authenticated, and once authenticated, are non-repudiable
-1. Statement provenance and history can be independently and consistently audited
-1. Issuers can efficiently prove that their Statement is logged by a Transparency Service
 
-The first guarantee is achieved by requiring Issuers to sign their Statements and associated metadata using a distributed public key infrastructure.
-The second guarantee is achieved by committing the Signed Statement to a Verifiable Data Structure.
-The third guarantee is achieved by the combination of both of these acts.
 
 ## Threat Model
 
 This section provides a generic threat model for SCITT, describing its residual security properties when some of its actors (Issuers, Transparency Services, and Auditors) are corrupt or compromised.
-
 This threat model may need to be refined to account for specific supply chain use cases.
 
 SCITT primarily supports checking of Signed Statement authenticity, both from the Issuer (authentication) and from the Transparency Service (transparency).
 These guarantees are meant to hold for extensive periods of time, possibly decades.
+Issuers and Transparency Services can both be compromised, which should be addressed with redundant and independent implementations.
 
-It can never be assumed that some Issuers and some Transparency Services will not be corrupt.
```

Comment:
Why are we comfortable removing this line ?

Reply:
This was considered duplicative text from line [917](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/370/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1R917)